pallets/token-manager/src/lib.rs

```diff
@@ -1092,4 +1096,8 @@ impl<T: Config> BridgeInterfaceNotification for Pallet<T> {
 
         Ok(())
     }
+
+    fn on_incoming_event_processed(event: &EthEvent) -> DispatchResult {
+        Self::processed_event_handler(event)
```

Comment:
Is it worth adding the feature guard here for `Lift` and `Growth` coming via this hook?

Reply:
No.
from the pallet point of view we either process events coming from eth-bridge or not.
The runtime defines the filter and the rest of the components are just executing.